presto-main/src/main/java/com/facebook/presto/type/RowType.java

```diff
@@ -67,17 +67,17 @@ public RowType(List<Type> fieldTypes, Optional<List<String>> fieldNames)
     public String getDisplayName()
     {
         // Convert to standard sql name
-        List<String> fields = new ArrayList<>();
-        for (int i = 0; i < this.fields.size(); i++) {
-            RowField field = this.fields.get(i);
+        List<String> fieldDisplayNames = new ArrayList<>();
```

Comment:
@cberner Updated to conform to the standard. I made the type name all lowercase to be consistent across all complex types, do we want to do that?

```
array<row(product_id bigint, timestamps varchar)>
```